<ui version="4.0" >
 <class>ScanParametersWidget</class>
 <widget class="QWidget" name="ScanParametersWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QWidget" native="1" name="scanParamsWidget" >
     <layout class="QVBoxLayout" name="verticalLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="fileLabel" >
          <property name="text" >
           <string>Spec File Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="specFileNameEdit" >
          <property name="minimumSize" >
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QLabel" name="scanTypeLabel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Scan type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="scanTypeComboBox" >
          <property name="frame" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="scanParametersLayout" />
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Integration Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="integrationTimeSpinBox" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>The integration time indicates how long to count at each point in the scan. 

Positive numbers correspond to seconds, negative numbers correspond to monitor counts.</string>
          </property>
          <property name="suffix" >
           <string/>
          </property>
          <property name="decimals" >
           <number>3</number>
          </property>
          <property name="minimum" >
           <double>-100000000.000000000000000</double>
          </property>
          <property name="maximum" >
           <double>1000000.000000000000000</double>
          </property>
          <property name="value" >
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="scanProgressFrame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="scanButton" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>0</y>
        <width>106</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text" >
       <string>Start Scan</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="scanProgressBar" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>130</x>
        <y>0</y>
        <width>301</width>
        <height>27</height>
       </rect>
      </property>
      <property name="contextMenuPolicy" >
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="value" >
       <number>0</number>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>17</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionPause" >
   <property name="text" >
    <string>Pause</string>
   </property>
   <property name="autoRepeat" >
    <bool>false</bool>
   </property>
   <property name="visible" >
    <bool>false</bool>
   </property>
  </action>
  <action name="actionResume" >
   <property name="text" >
    <string>Resume</string>
   </property>
   <property name="autoRepeat" >
    <bool>false</bool>
   </property>
   <property name="visible" >
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAbort" >
   <property name="text" >
    <string>Abort</string>
   </property>
   <property name="autoRepeat" >
    <bool>false</bool>
   </property>
   <property name="visible" >
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
